src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -25,17 +25,6 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
 
   def adminToken = HttpGoogleServicesDAO.getAdminUserAccessToken
 
-  override def sendNotifications(notifications: Seq[Notification]): Future[Try[Unit]] = {
-
-    val notificationPipeline = addCredentials(OAuth2BearerToken(adminToken)) ~> addHeader(fireCloudHeader) ~> sendReceive
-    val thurloeNotifications = notifications.map(n => ThurloeNotification(n.userId, n.userEmail, n.replyTo, n.notificationId, n.toMap))
```

Comment:
also remove the `ThurloeNotification` case class from `Profile.scala` and the json implicit from `ModelJsonProtocol.scala`